blueocean-pipeline-api-impl/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.jenkins.blueocean</groupId>
         <artifactId>blueocean-parent</artifactId>
-        <version>1.5.1-SNAPSHOT</version>
+        <version>1.5.1-beta-1-SNAPSHOT</version>
```

Comment:
@kzantow Not sure this change alone will work. Perhaps you want  set beta-1 to all the poms. You can do the following from project root directory:

```
$ mvn versions:set -DnewVersion=1.5.1-beta-1-SNAPSHOT
$ mvn versions:commit
```